packages/protocol/contracts/team/airdrop/MerkleClaimable.sol

```diff
@@ -59,7 +76,15 @@ abstract contract MerkleClaimable is EssentialContract {
         }
 
         isClaimed[hash] = true;
-        _claimWithData(data);
```

Comment:
Seems a bit like a strange setup where the claiming has an extra internal function to extend the functionality. More logical to me would be to have an external `claimAndDelegate()` function (or something like that) that would call `claim` and also calls `delegateBySig()`.

Reply:
I think from the op's perspective, we want to make sure if some one is in the airdrop list, the address must claim by itself, no way can claim for it, so we filter out dead addresses. And we also want to make sure whenever you claim, you must delegate to someone.

Reply:
However, the extra function shall go to ERC20Aridrop contracts, not in this parent contract.